notebooks/image_models/solutions/vertex_tuning_image_classification.ipynb

```diff
@@ -0,0 +1,1225 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    !gcloud builds submit --machine-type=e2-highcpu-32 --timeout=15m --tag $TRAINING_CONTAINER_IMAGE_URI flower_trainer
</pre><p>Just out of curiosity, but do we need to use this large machine to build this?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/225/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='30'/>

Reply:
<p>Just to make the build a little faster so there's less waiting in the lab</p>

Reply:
<p>I had the same question, what's the decrease in training time with using this machine?</p>